### ISSUE TITLE
Search (or filter / sort) feature for data dictionary

### ISSUE THREAD

**Issue by lzachmann** [askiggy/docs#12]
- At the very bottom of the ["Intro to Iggy Place Data" page](https://docs.askiggy.com/reference/place-data), it says:
> The Data Dictionary is searchable by data source, attribute, aggregation, and normalization as well as feature name.
But when I go to the [data dictionary](https://docs.askiggy.com/reference/data-dictionary) I'm not seeing a search bar of any sort. Perhaps sign-in is required?

**Comment by bdejesus** [comment]
🤔 I don't think we ever had an actual "search" feature for the data dictionary (unless ctrl+f counts) – it originally was shared as a google spreadsheet. We've also discussed building a more searchable/browsable "Data Catalog" page, but it's currently been de-prioritized. Maybe for now we can just reword that sentence to be more accurate to the content format.

**Comment by lzachmann** [comment]
Agreed. Maybe it's as easy as: "The Data Dictionary [link to page] includes information about data sources, attributes, aggregation, and normalization as well as feature name." Or something like that?
